src/olympia/lib/settings_base.py

```diff
@@ -1656,6 +1656,9 @@ def get_redis_settings(uri):
 
     # Use json by default when using APIClient.
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
+
+    # Use http://ecma-international.org/ecma-262/5.1/#sec-15.9.1.15
```

Comment:
added in https://github.com/mozilla/addons-server/pull/3741/commits/2fd89bdb4a46367b9c624fbd2f0a13f93fb641d7